cluster/charts/rook-ceph/templates/resources.yaml

```
@@ -1140,6 +1140,14 @@ spec:
                       enum:
                         - image
                         - pool
+                    snapshotSchedule:
```

Looks like the schema needs to be updated in several places
```suggestion
                    snapshotSchedules:
```